odahuflow/airflow_plugin/connection.py

```diff
@@ -14,39 +14,39 @@
 #    limitations under the License.
 #
 
-from airflow.contrib.hooks.gcp_sql_hook import GCP_CREDENTIALS_KEYFILE_DICT
 from airflow.models import BaseOperator, Connection as AirflowConnection
 from airflow.utils.decorators import apply_defaults
-from legion.sdk.clients.connection import ConnectionClient
-from legion.sdk.clients.edi import WrongHttpStatusCode
-from legion.sdk.models import Connection
+from odahuflow.airflow_plugin.api import OdahuHook
+from odahuflow.sdk.clients.api import WrongHttpStatusCode
+from odahuflow.sdk.clients.connection import ConnectionClient
+from odahuflow.sdk.models import Connection
 
-from legion.airflow.edi import LegionHook
+GCP_CREDENTIALS_KEYFILE_DICT = "extra__google_cloud_platform__keyfile_dict"
```

Comment:
Why don't you import the constant from the contrib package?

Reply:
Because its optional package that requires mySQL binaries, so I think it's not a good deal to require airflow[mysql] package only for this constant (It doesn't defined in other modules)